core/gallery/models.go

```diff
@@ -112,9 +113,16 @@ func InstallModel(basePath, nameOverride string, config *Config, configOverrides
 		if err := utils.VerifyPath(file.Filename, basePath); err != nil {
 			return err
 		}
+
 		// Create file path
 		filePath := filepath.Join(basePath, file.Filename)
 
+		scanResults, err := downloader.HuggingFaceScan(file.URI)
```

Comment:
this is what I'm a bit against. I see that we are not touching base to HF unless it's a HF URL, but things might go wrong in different ways here, like, for example HF having API issues would bite us here and prevent pulling a file , for instance:

```
		return nil, fmt.Errorf("unexpected status code during HuggingFaceScan: %d", results.StatusCode)
``
could just file for some unrelated reason. 

Can we just drop this? or have it behind a flag ?

Reply:
I understand the risk there. I'm happy to put it behind a flag, but I'd really like to _enable_ said flag by default - if a temporary HF API outage happens that _doesn't_ also take down their fileservers, we can post heavily in discord / document on the site a temporary flag to bypass the safety valve.

Reply:
that's reasonable :+1: works for me !